src/DependencyInjection/SonataMediaExtension.php

```diff
@@ -291,11 +291,11 @@ public function configureFilesystemAdapter(ContainerBuilder $container, array $c
 
             $container->getDefinition('sonata.media.metadata.amazon')
                 ->replaceArgument(0, [
-                        'acl' => $config['filesystem']['s3']['acl'],
-                        'storage' => $config['filesystem']['s3']['storage'],
-                        'encryption' => $config['filesystem']['s3']['encryption'],
-                        'meta' => $config['filesystem']['s3']['meta'],
-                        'cache_control' => $config['filesystem']['s3']['cache_control'],
+                    'acl' => $config['filesystem']['s3']['acl'],
```

Comment:
Why is this not caught by the linter ? Is there a rule for this ?

Reply:
Not sure, I think this is not the first time I see something like this in Sonata, copy pastes bad indented.